III.tdd-katas/fizz-buzz/fizz-buzz.test.js

```diff
@@ -2,10 +2,10 @@ var fizzbuzz = require('./fizz-buzz')
 
 test('prints 1', function () {
     var result = fizzbuzz(1)
-    expect(result).toEqual('1')
+    expect(result).toEqual(1)
```

Comment:
woops! We have changed the tests to conform to our implementation here.
Can we keep the tests intact, that is, always returning String?
Perhaps there is a way to convert numbers to String in JS :+1:

Reply:
I tried to use .toString() method, but I stuck

Reply:
I update solution